src/safir/sentry/_helpers.py

```diff
@@ -83,3 +87,42 @@ def before_send_handler(event: Event, hint: Hint) -> Event:
     """
     event = fingerprint_env_handler(event, hint)
     return sentry_exception_handler(event, hint)
+
+
+async def notify(
```

Comment:
`notify` is pretty generic, particularly when it's been imported as a bare function. Maybe `report_exception` instead? (Sorry, I should have noticed that in my previous review before you moved it and it would have saved some work.)